### PR TITLE
scan-analytics

### PR DESCRIPTION
**Pull request by AzgatG**
-entities
-modules
-logic

### CODE REVIEW

File: src/accounts/accounts.service.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  ) {}
+
+  @Cron('25 * * * *')
+  async accountsCron() {
+    const chains = this.appConfigService.CHAINS;
+    for (const { url, name } of chains) {
+      try {
+        await this.stats(url, name);
+      } catch (e) {
+        continue;
```
Comment (chernetsky) [inline src/accounts/accounts.service.ts]: Maybe log this error?

File: src/app.module.ts [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
+  imports: [
+    ConfigModule.forRoot({ validate: envValidate }),
+    TypeOrmModule.forRoot({
+      ...typeormConfig,
+      entities: [
+        AccountsStats,
+        TokensStats,
+        TransfersStats,
+        CollectionsStats,
+        TransfersStats,
```
Comment (chernetsky) [inline src/app.module.ts]: Remove duplicate entity.

File: src/transfers/transfers.service.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import { getExtrinsicsStat } from './transfers.queries';
+import { TransfersStats } from '../../common/entities/TransfersStats';
+import { GqlService } from '../../common/gqlService';
+import { AppConfigService } from '../utils/app-config.service';
+import { UtilsService } from '../utils/utils.service';
+
+enum ExtrinsicsStatsTypeEnum {
+  COINS = 'COINS',
+  TOKENS = 'TOKENS',
+  TRANSACTIONS = 'TRANSACTIONS',
```
Comment (chernetsky) [inline src/transfers/transfers.service.ts:14]: Why do we have three types in this repo, but only two in `unique-scan-backend` repo (COINS and TOKENS)? TRANSACTIONS means everything?

File: src/utils/app-config.service.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    return this.configService.get<string>('UNIQUE_CHAIN_GQL_API')!;
+  }
+
+  get QUARTZ_CHAIN_GQL_API(): string {
+    return this.configService.get<string>('QUARTZ_CHAIN_GQL_API')!;
+  }
+
+  get CHAINS() {
+    return [
+      { name: 'OPAL', url: this.OPAL_CHAIN_GQL_API },
```
Comment (chernetsky) [inline src/utils/app-config.service.ts]: Let's move string literals into constant enum!:)

**Review by chernetsky — CHANGES_REQUESTED** [review]
There are some minor comments. But you did a great job!)

File: src/transfers/transfers.service.ts (hunk 3 above)
Comment (AzgatG) [reply src/transfers/transfers.service.ts:14]: yep

File: src/accounts/accounts.service.ts [hunk 5]
```diff
@@ -21,6 +21,8 @@ interface IAccountsResponse {
 
 @Injectable()
 export class AccountsService {
+  private readonly logger = new Logger('AccountsService');
```
Comment (chernetsky) [inline src/accounts/accounts.service.ts:24]: This we can do like this `private readonly logger = new Logger(AccountsService.name);`

**Review by chernetsky — APPROVED** [review]

**Pull request merged**